packages/blocks/src/__internal__/debug-menu.ts

```diff
@@ -141,18 +141,18 @@ export class DebugMenu extends LitElement {
   @state()
   _mode: 'page' | 'edgeless' = 'page';
 
-  get space() {
-    return this.store.space;
-  }
+  // get space() {
+  //   return this.store.space;
+  // }
 
   private _onToggleConnection() {
-    if (this.connected === true) {
-      this.store.providers.forEach(provide => provide.disconnect());
-      this.connected = false;
-    } else {
-      this.store.providers.forEach(provide => provide.connect());
-      this.connected = true;
-    }
+    // if (this.connected === true) {
```

Comment:
I think this should be enabled

Reply:
It will be reopen soon, debug-menu will be moved to the editor file